sdk/communication/communication-phone-numbers/review/communication-phone-numbers.api.md

```diff
@@ -113,6 +113,7 @@ export type PhoneNumberAssignmentType = "person" | "application";
 export interface PhoneNumberCapabilities {
```

Comment:
I just noticed that `PhoneNumberCapabilities` and `PhoneNumberCapabilitiesRequest` are identical. We probably should use only one of them.